docs/overview.rst

```diff
@@ -0,0 +1,45 @@
+Overview
+========
+
+
+Start your own Vyper project
+----------------------------
+
+Twig lets you write tests using ``web3.py`` and ``pytest-ethereum`` against your ``Vyper`` smart contracts.
+
+
+Twig setup
+~~~~~~~~~~
+
+- clone twig repo
+- ``python -m venv venv`` (note: requires python3.6)
+- ``. venv/bin/activate``
+- ``pip install -e .``
+
+
+Project setup
+~~~~~~~~~~~~~
+
+.. NOTE:: ``twig/contracts/name_registry.vy`` and ``tests/contracts/test_name_registry.py`` have been included as examples on how to get started, but are safe to delete.
+
+- create your vyper contract file in ``twig/contracts/``
+  (i.e. ``twig/contracts/example.vy``)
+
+- write your vyper contract test file in ``tests/contracts/``
+  (i.e. ``twig/contracts/test_example.py``)
+
+- create your contract's ``Deployer`` fixture in your test file.
```

Comment:
This is confusing to me.  Is the code below creating the `Deployer` fixture?  If so I'm confused because neither of the fixtures in the subsequent code block are named `Deployer`.